### PR TITLE
[sonic-cfggen] Add check to generate valid bgpconfig

### PR DESCRIPTION
**Pull request by NStetskovych**
Trying to apply manual bgp config and
loading minigraph with no bgp configuration leads to bgp daemon failed to start.
Log attached
[bgp_status_log.txt](https://github.com/Azure/sonic-buildimage/files/871122/bgp_status_log.txt)
.

Signed-off-by: Nadiya.Stetskovych <Nadiya.Stetskovych@cavium.com>

### CODE REVIEW

File: dockers/docker-fpm/bgpd.conf.j2 [hunk 1]
```diff
@@ -14,6 +14,7 @@ log facility local4
 ! enable password {# {{ en_passwd }} TODO: param needed #}
 {% endblock system_init %}
 !
+{% if minigraph_bgp_asn != None %}
```
Comment (pavel-shirshov) [inline dockers/docker-fpm/bgpd.conf.j2]: Will the following work better for Jinja2?
{% if minigraph_bgp_asn is defined %}

File: dockers/docker-fpm/bgpd.conf.j2 [hunk 2]
```diff
@@ -57,8 +59,10 @@ router bgp {{ minigraph_bgp_asn }}
 {% endfor %}
 {% endblock bgp_sessions %}
 !
+{% if minigraph_bgp_asn != None %}
```
Comment (pavel-shirshov) [inline dockers/docker-fpm/bgpd.conf.j2]: The same as above

File: dockers/docker-fpm/bgpd.conf.j2 (hunk 1 above)
Comment (NStetskovych) [inline dockers/docker-fpm/bgpd.conf.j2]: hmm It`s defined to None :). Have no effect.

Comment (pavel-shirshov) [inline dockers/docker-fpm/bgpd.conf.j2]: Yes. You're right. But it's unusual to see something != None in python. Usually we write something is not None.
I don't know the right patter for Jinja2.
I found there're 'none' constant is Jinja2 templates http://jinja.pocoo.org/docs/2.9/templates/#expressions

Also I found a special function none() (see the same document).

So my concert here: I don't want to have an expression in our code which might be deprecated.

**Pull request merged**